.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+Thanks for submitting a Pull Request!
+
+Please shortly explain your contribution, and if fixing an issue from the tracker, please add "Fixes #XXX", replacing XXX with the issue number.
+
+Be ensure that your contribution follows the [Developer Guide](https://index.ros.org/doc/ros2/Contributing/Developer-Guide/).
+
+Be sure to go over each item in the list below before submitting your pull request.
+-->
+
+### Description
+
+Add your description here
```

Comment:
```suggestion
Add your description here
```

Briefly describe what your changes are doing.  It is more important to mention why the changes are necessary, rather than the how.